client/src/app/services/auth.service.ts

```diff
@@ -87,4 +89,25 @@ export class AuthService {
     };
     return this.backend.editCell(payload);
   }
+
+  SendGlobalMessage(
+    username: string,
+    message: string
+  ) : Observable<never> {
+    const payload: SendGlobalMessageRequest = {
+      username,
+      message
+    };
+    return this.backend.sendGlobalMessage(payload);
+  }
+
+  GetGlobalMessages(): Observable<Message[]> {
+    return this.backend.getGlobalMessages().pipe(
+      map((response) => 
+        response.messages
+      )
+    );
+  }
 }
+
+  
```

Comment:
Please fix this and other formatting issues (missing newlines at EOF, trailing whitespace and inappropriate line breaking).